### PR TITLE
[WIP] Add tasks to get detail about node not ready

### PR DESCRIPTION
**Pull request by mtnbikenc**

### CODE REVIEW

File: roles/openshift_node/tasks/config.yml [hunk 1]
```diff
@@ -213,16 +213,33 @@
...
+  - name: Gather oc get node output
+    command: >
+      oc get node {{ hostvars[item].ansible_nodename | lower }}
+      --config={{ openshift_node_kubeconfig_path }}
+      --output=jsonpath='{.status.conditions[?(@.type=="Ready")].status}'
+    loop: "{{ ansible_play_batch }}"
+    delegate_to: localhost
+    run_once: true
+    register: oc_get
+  - name: Print oc get node output
```
Comment (vrutkovs) [inline roles/openshift_node/tasks/config.yml]: That would only output smth like `NotReady Ready Ready` afaik. Perhaps its worth running `oc get nodes -o yaml` to get full node output instead?

Comment (mtnbikenc) [reply roles/openshift_node/tasks/config.yml]: I forgot to change that part.  Oof...

**Pull request closed without merging**